src/fsharp/FSharp.Compiler.Private/FSharp.Compiler.Private.fsproj

```diff
@@ -696,6 +695,11 @@
 
   <ItemGroup Condition="'$(TargetFramework)' == 'net472'">
     <Reference Include="ISymWrapper, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" />
+    <Reference Include="System.Collections" />
```

Comment:
same story here for these `Reference` nodes. Should they be strongly-referenced and/or is there a package equivalent?